dex/testing/loadbot/mantle.go

```diff
@@ -467,8 +547,60 @@ func newBotWallet(symbol, node, name string, pass []byte, minFunds, maxFunds uin
 		}
 	case btc:
 		form = &core.WalletForm{
-			Type:    "bitcoindRPC",
+			Type:    "SPV",
 			AssetID: btcID,
+			Config: map[string]string{
+				"rpcport": rpcAddr(symbol, node),
```

Comment:
Two questions: Shouldn't we be able to test with bitcoindRPC?  In SPV, what's this `rpcport` setting for?

Reply:
> Shouldn't we be able to test with bitcoindRPC?

Yes. I just thought that most users will be using SPV so it's better to "test" with that. Will change to RPC tho.

> In SPV, what's this rpcport setting for?

Not needed...

Reply:
> Will change to RPC tho

Well if I had to pick one to use on loadbot it would depend on what I'm hoping to test.  For client I agree SPV is best.  If aiming to just load the server, client might as well be super fast with full node.

Just thinking aloud

Reply:
Change to rpc.